src/libs/dutil/dirutil.cpp

```diff
@@ -349,6 +349,37 @@ extern "C" HRESULT DAPI DirEnsureDeleteEx(
 
 
 /*******************************************************************
+DirDeleteEmptyDirectoriesToRoot - removes an empty directory and as many
+                                  of its parents as possible.
+
+ Returns: count of directories deleted.
+*******************************************************************/
+extern "C" DWORD DAPI DirDeleteEmptyDirectoriesToRoot(
+    __in_z LPCWSTR wzPath,
+    __in DWORD /*dwFlags*/
+    )
+{
+    DWORD cDeletedDirs = 0;
+    LPWSTR sczPath = NULL;
+
+    while (wzPath && *wzPath && ::RemoveDirectoryW(wzPath))
```

Comment:
Does it make since to at least log result from GetLastError if RemoveDirectoryFails for any error other than an expected error/?